co-simulation tool/bundle/src/assembly/resources/etc/runtime.json

```diff
@@ -0,0 +1,242 @@
+            "configuration": "cell_config.json",
+            "priority": 50,
+            "host": "local",
+            "port": 0,
+            "deploy": false,
+            "start": false,
+            "subscriptions": [
+                "VehicleRegistration",
+                "RsuRegistration",
+                "TmcRegistration",
+                "ServerRegistration",
+                "TrafficLightRegistration",
+                "ChargingStationRegistration",
+                "VehicleUpdates",
+                "V2xMessageTransmission",
+                "CellularCommunicationConfiguration"
+            ],
+            "javaClasspathEntries": []
+        },
+        {
+            "id": "omnetpp",
+            "classname": "org.eclipse.mosaic.fed.omnetpp.ambassador.OmnetppAmbassador",
+            "configuration": "omnetpp_config.json",
+            "configurationDeployPath": "omnetpp-federate/simulations",
+            "priority": 50,
+            "dockerImage": "",
+            "host": "local",
+            "port": 4998,
+            "deploy": true,
+            "start": true,
+            "subscriptions": [
+                "VehicleRegistration",
+                "RsuRegistration",
+                "TrafficLightRegistration",
+                "VehicleUpdates",
+                "V2xMessageTransmission",
+                "AdHocCommunicationConfiguration"
+            ],
+            "javaClasspathEntries": []
+        },
+        {
+            "id": "ns3",
+            "classname": "org.eclipse.mosaic.fed.ns3.ambassador.Ns3Ambassador",
+            "configuration": "ns3_config.json",
+            "configurationDeployPath": "ns3/scratch",
+            "priority": 50,
+            "dockerImage": "",
+            "host": "local",
+            "port": 5011,
+            "deploy": true,
+            "start": true,
+            "subscriptions": [
+                "VehicleRegistration",
+                "RsuRegistration",
+                "TrafficLightRegistration",
+                "VehicleUpdates",
+                "V2xMessageTransmission",
+                "AdHocCommunicationConfiguration"
+            ],
+            "javaClasspathEntries": []
+        },
+        {
+            "id": "sns",
+            "classname": "org.eclipse.mosaic.fed.sns.ambassador.SnsAmbassador",
+            "configuration": "sns_config.json",
+            "priority": 50,
+            "host": "local",
+            "port": 0,
+            "deploy": false,
+            "start": false,
+            "subscriptions": [
+                "VehicleRegistration",
+                "RsuRegistration",
+                "TrafficLightRegistration",
+                "VehicleUpdates",
+                "V2xMessageTransmission",
+                "AdHocCommunicationConfiguration"
+            ],
+            "javaClasspathEntries": []
+        },
+        {
+            "id": "sumo",
+            "classname": "org.eclipse.mosaic.fed.sumo.ambassador.SumoGuiAmbassador",
```

Comment:
Is SumoAmbassador used in the co-simulation tool? There are two new parameters receivedSimulationStep and sumoCarlaCoSimulation in SumoAmbassador class.

Reply:
SumoGuiAmbassador class is a child class of SumoAmbassador class. When the users want to use SUMO GUI, the users need to specify the classname as org.eclipse.mosaic.fed.sumo.ambassador.SumoGuiAmbassador in runtime.json. I set the co-simulation tool starts SUMO GUI by default.